src/Infrastructure/MembershipApplicationTrackingInfo.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class MembershipApplicationTrackingInfo {
+
+	private $totalImpressionCount;
+	private $singleBannerImpressionCount;
+	private $confirmationPage;
+	private $confirmationPageCampaign;
```

Comment:
It's generated when confirming a donation and needs to be put into that embedded membership application form. Since I've been working a lot around the membership forms, I can do that.